src/typings/global.d.ts

```diff
@@ -34,3 +35,13 @@ declare module "express" {
     locals: ArtsyResponseLocals
   }
 }
+
+declare module "styled-components" {
```

Comment:
By the way, the `ServerStyleSheet` type definition has been merged. https://github.com/styled-components/styled-components/pull/839 so we can remove this